brainstormforce/astra
# AST-1965 - Fix css specificity

### Description

- Adjusted CSS Specificity in woocommerce-grid.css
- Adjusted CSS Specificity in woocommerce.css

### Screenshots
NA

### Types of changes


- Improved css specificity





### How has this been tested?


- Updated CSS should work as expected like current one is working
- Check WooCommerce pages 
- Check with Legacy & modern Ecommerce layouts of Astra

### Checklist:
